snowpack/src/sources/util.ts

```diff
@@ -16,19 +16,25 @@ export async function clearCache() {
   ]);
 }
 
-/**
- * Previously, all PackageSources were global. The new PackageSourceLocal is designed
- * to allow for uniqueness across projects / Snowpack instances. That support will come later,
- * so for now we just keep a global instance here.
- */
-let sharedPackageSourceLocal: PackageSourceLocal | undefined;
-let sharedPackageSourceRemote: PackageSourceRemote | undefined;
+const remoteSourceCache = new WeakMap<SnowpackConfig, PackageSource>();
+const localSourceCache = new WeakMap<SnowpackConfig, PackageSource>();
```

Comment:
Ooh is this a WeakMap appearance for Snowpack? Exciting!

Reply:
Probably should use these more, TBH. AFAIK we could benefit from GC in more places

Reply:
Should have called it strongmap. 💪